shell/platform/linux/fl_task_runner.cc

```diff
@@ -24,7 +24,7 @@ struct _FlTaskRunner {
 typedef struct _FlTaskRunnerTask {
   // absolute time of task (based on g_get_monotonic_time)
   gint64 task_time_micros;
-  FlutterTask task;
+  std::function<void()> delegate;
```

Comment:
`FlTaskRunnerTask` is freed using `g_free`. The implicit c++ destructor is not called at any point. So you'll leak any heap allocation that `std::function` might have. Again, it would be better to just have a callback and gpointer instead.